.circleci/config.yml

```diff
@@ -23,19 +23,24 @@ jobs:
           name: Install dependencies
           command: |
             pip install --upgrade pip twine wheel
-            pip install --editable .[test] --upgrade
+            pip install --editable .[test,lint] --upgrade
```

Comment:
Why not put the linter with the test dependencies ?

Reply:
Pour apporter de la granularité aux niveaux des dépendances.

Reply:
Dans quel cas concret cette granularité est-elle souhaitable?
Le package `yamllint ` est-il connu pour être difficile à installer dans certains environnements?
S'agit-il d'un package très lourd qui ralentit significativement l'installation?

Un linter me paraît être typiquement une dépendance de développement, au même titre  que nose ou flake8. S'il n'y a pas de raison particulière, je suis pour qu'on le mette dans `test`, avec les autres dépendances de dev.

(Il y a déjà bien trop d'extra-dépendances sur France.)